apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -399,15 +402,15 @@ export class SubscribersController {
     return (await this.getPreferenceUsecase.execute(command)) as UpdateSubscriberPreferenceResponseDto[];
   }
 
-  @Get('/:subscriberId/preferences/:level')
+  @Get('/:subscriberId/preferences/:parameter')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** I'm not sure why we need to change this one?

Reply:
it failed openApi validation, it's an illegal behavior, path params should have the same name